tee-worker/litentry/core/identity-verification/src/web2/mod.rs

```diff
@@ -98,18 +98,20 @@ pub fn verify(
 				.get_user_info(message.author.id.clone())
 				.map_err(|e| Error::LinkIdentityFailed(e.into_error_detail()))?;
 
-			let mut user_name_with_discriminator = message.author.username.clone();
-			user_name_with_discriminator.push_str(&'#'.to_string());
-			user_name_with_discriminator.push_str(&user.discriminator);
-
+			let mut user_name = message.author.username.clone();
+			// if discord user's username is upgraded complete, the discriminator value from api will be "0".
```

Comment:
Maybe it would be useful to have an issue for removing this special handling once the upgrade is completed

Reply:
Probably; I have a feeling it's gonna be a looong time before we can yeet it tho ;)